src/app/InteractionModelEngine.cpp

```diff
@@ -1370,19 +1371,14 @@ bool InteractionModelEngine::HasActiveRead()
     }) == Loop::Break));
 }
 
-uint16_t InteractionModelEngine::GetMinSubscriptionsPerFabric() const
+uint16_t InteractionModelEngine::GetMinGuaranteedSubscriptionsPerFabric() const
 {
-    uint8_t fabricCount                  = mpFabricTable->FabricCount();
-    const size_t readHandlerPoolCapacity = GetReadHandlerPoolCapacityForSubscriptions();
-
-    if (fabricCount == 0)
-    {
-        return kMinSupportedSubscriptionsPerFabric;
-    }
-
-    size_t perFabricSubscriptionCapacity = readHandlerPoolCapacity / fabricCount;
-
-    return static_cast<uint16_t>(perFabricSubscriptionCapacity);
+#if CHIP_SYSTEM_CONFIG_POOL_USE_HEAP
+    return UINT16_MAX;
```

Comment:
This would then claim that the device guarantees this many subscriptions.  But it may not actually be able to handle this many...

Followup is ok, but we need to figure out something saner here.

Reply:
Sure, but what about be saner?

Reply:
If I had a concrete suggestion I would have made it....  Hence followup.